commands/create_cluster.go

```diff
@@ -167,8 +167,6 @@ func init() {
 	CreateClusterCommand.Flags().MarkDeprecated("kubernetes-version", "please use --release to specify a release to use")
 	CreateClusterCommand.Flags().MarkDeprecated("num-workers", "please use --workers-min and --workers-max to specify the node count to use")
 
-	CreateClusterCommand.MarkFlagRequired("owner")
```

Comment:
The validation is done below already. There is no reason to use this feature here, especially because it prevents the owner being configured via config files. This works now again.